examples/piecestore-client/main.go

```diff
@@ -140,7 +140,7 @@ func main() {
 				return err
 			}
 
-			rr, err := psClient.Get(ctx, client.PieceID(id), pieceInfo.Size, &pb.PayerBandwidthAllocation{})
+			rr, err := psClient.Get(ctx, client.PieceID(id), pieceInfo.Size, &pb.PayerBandwidthAllocation{}, &pb.SignedMessage{})
```

Comment:
Can't we pass `nil` instead of `&pb.SignedMessage{}`?

Reply:
sure, done